Grizzly-Trolls/welcome-to-nashville
# Sc event listener

git fetch --all
git checkout sc-event-listener

# Description

Added drop down menu to search page 
Please check this is working

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Run server and open code in browser
check that drop down menu is working and that results are appearing in the DOM for each selection item.


# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
